src/components/Button/Button.js

```diff
@@ -38,101 +36,14 @@ function Button({
   shadow, // boolean
   ...rest
 }) {
-
-  console.log('Button')
-
-
-  const chooseButtonClasses = () => {
-    // type, gradient, border, shape, shadow, size
-    let buttonType = type || ButtonTypesEnum.primary;
-
-    let buttonStyling = '';
-    buttonStyling += shadow ? 'shadow' : '';
-
-
-    switch(type) {
-      case ButtonTypesEnum.success:
-        if (hollow) {
-          buttonStyling += ' hollow hollow-success border'
-        } else {
-          buttonStyling += ' text-white';
-          buttonStyling += gradient ? ' success-gradient' : ' success';
-          buttonStyling += border ? ' border' : ' zero-border';
-        }
-        break;
-      case ButtonTypesEnum.warning:
-        if (hollow) {
-          buttonStyling += ' hollow hollow-warning border'
-        } else {
-          buttonStyling += ' text-white';
-          buttonStyling += gradient ? ' warning-gradient' : ' warning';
-          buttonStyling += border ? ' border' : ' zero-border';
-        }
-        break;
-      case ButtonTypesEnum.danger:
-        if (hollow) {
-          buttonStyling += ' hollow hollow-danger border'
-        } else {
-          buttonStyling += ' text-white';
-          buttonStyling += gradient ? ' danger-gradient' : ' danger';
-          buttonStyling += border ? ' border' : ' zero-border';
-        }
-        break;
-      case ButtonTypesEnum.disabled:
-               if (hollow) {
-          buttonStyling += ' hollow hollow-disabled border'
-        } else {
-          buttonStyling += ' text-white';
-          buttonStyling += gradient ? ' disabled-gradient' : ' disabled';
-          buttonStyling += border ? ' border' : ' zero-border';
-        }
-        break;
-      default: // primary
-        if (hollow) {
-          buttonStyling += ' hollow hollow-primary border'
-        } else {
-          buttonStyling += ' text-white';
-          buttonStyling += gradient ? ' primary-gradient' : ' primary';
-          buttonStyling += border ? ' border' : ' zero-border';
-        }
-    }
-
-    // shape, // enum: box(defualt), rounded, pill
-    switch(shape) {
-      case ButtonShapesEnum.rounded:
-        buttonStyling += ' rounded';
-        break;
-      case ButtonShapesEnum.pill:
-        buttonStyling += ' pill';
-        break;
-      default: // box
-    }
-
-    // size, // enum: small, medium, big
-    switch(size) {
-      case ButtonSizesEnum.small:
-        buttonStyling += ' small';
-        break;
-      case ButtonSizesEnum.big:
-        buttonStyling += ' big';
-        break;
-      default: // medium
-        buttonStyling += ' medium';
-    }
-
-    return buttonStyling;
-
-  }
-
-  // ButtonTypesEnum
-
-
   return (
-    <button {...rest} className={chooseButtonClasses()}>
+    <button
```

Comment:
This styling is good for this particular instance of a button, but is not generalizable enough for a Button component.  If you don't intend to reuse this, then could put this styling right on the homepage.

Reply:
will tackle i a new PR